communication.md

```diff
@@ -0,0 +1,37 @@
+# Communication
+
+In Discord, we have categories, text and voice channels for communication
+purposes as well.
+
+## Public Category
+
+Anything under this category is read/write for a Contributor and above, but
+read-only for the rest.
+
+It is allowed to open various text and voice channels under this category.
+
+Voice channels under this category should restrict users to use Push-to-Talk.
+
+## Meet Category
+
+Meet category is to allow organize meetings in physical and digital worlds.
+
+Text channels under this category;
+
+- Can be created, updated and deleted according to the project management and
+  communication needs
+- Are member only, and further restriction can be made according to functional
+  roles
+
+Voice Channels under this category;
+
+- Don't restrict voice activity but users should be limited.
```

Comment:
It's not clear. Revise like this. "Can be used with voice activity setting enabled in discord. By the nature of usage number of users should be limited." instead